src/utils/graphql/isFeatureEnabled.tsx

```diff
@@ -0,0 +1,60 @@
+import { FeatureFlag } from 'components/core/enums';
+import { readInlineData, graphql } from 'relay-runtime';
+import isProduction from 'utils/isProduction';
+import { isFeatureEnabledFragment$key } from '__generated__/isFeatureEnabledFragment.graphql';
+
+const PROD_FLAGS: Record<FeatureFlag, boolean> = {
+  GENERAL_MEMBERSHIP_MINT: true,
+  POSTER_MINT: false,
+};
+
+const DEV_FLAGS: Record<FeatureFlag, boolean> = {
+  GENERAL_MEMBERSHIP_MINT: true,
+  POSTER_MINT: true,
+};
+
+const EMPLOYEE_FLAGS: Record<FeatureFlag, boolean> = {
+  GENERAL_MEMBERSHIP_MINT: true,
+  POSTER_MINT: true,
+};
+
+const EMPLOYEE_USER_IDS = new Set(
+  process.env.NEXT_PUBLIC_EMPLOYEE_IDS ? process.env.NEXT_PUBLIC_EMPLOYEE_IDS.split(',') : []
+);
```

Comment:
added a list of employee IDs to our environment variables via vercel. this is also available in a notion doc